src/components/Contacts/ContactView/StatusSelector/StatusSelector.tsx

```diff
@@ -0,0 +1,79 @@
+import { Box, InputLabel, MenuItem, Select } from '@material-ui/core';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import { ErrorOutline } from '@material-ui/icons';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  statusLabel: {
+    padding: theme.spacing(0),
+    fontFamily: theme.typography.fontFamily,
+    color: theme.palette.text.primary,
+    fontSize: '12px',
+    letterSpacing: '0.4px',
+    margin: '4px 0px',
+  },
+  statusMenuItem: {
+    fontFamily: theme.typography.fontFamily,
+    fontSize: '16px',
+    background: theme.palette.common.white,
+  },
+  statusIcon: {
+    color: theme.palette.secondary.main,
+    marginRight: theme.spacing(1),
+    width: '20px',
+    height: '20px',
+  },
+  statusSelector: {
+    fontFamily: theme.typography.fontFamily,
+    fontSize: '16px',
+    background: theme.palette.common.white,
+    paddingLeft: '16px',
+    paddingRight: '16px',
+    paddingTop: '8px',
+    paddingBottom: '8px',
+  },
+}));
+interface StatusSelectorProps {
+  statuses: string[];
```

Comment:
I created https://jira.cru.org/browse/MPDX-6958 to load these. It seemed like a good first custom hook/GraphQL ticket for you so I assigned it to you. Hopefully this component could call it like `const { statuses } = useApiConstants();`. I'd be happy to work with you on it if you need more direction. You can create a `.graphql` file with the query and then run `yarn gql` which should generate a `useLoadConstantsQuery()` hook for you that you can call in this new custom hook.

I'm fine if you want to merge this PR as is first though and come back to this.

Seems like a cleaner way than making something else load them and pass them in.